recipes/asdf-unit-schemas/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "asdf-unit-schemas" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asdf_unit_schemas-{{ version }}.tar.gz
+  sha256: 42b78d67213efe4ffd4529fb0e58d9c7a0dab5cbf8839b230f1bc0a446bff999
+
+build:
+  skip: true  # [py<38]
```

Comment:
```suggestion
  noarch: python
```

Reply:
The linter complained that selectors cannot be used for noarch packages. Is there a way to handle the version dependency for importlib-resources without using a selector?
With [this version](https://github.com/conda-forge/staged-recipes/blob/ddefe581fcdf8a8f7d5acfe8159c4c87790c87f2/recipes/asdf-unit-schemas/meta.yaml) the linter reported: noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.
(in addition to errors for the python version having spaces).

Reply:
We generally just unconditionally install things like importlib-resources in this case.